code/modules/mob/living/simple_animal/hostile/venus_human_trap.dm

```diff
@@ -55,21 +61,19 @@
 
 /obj/structure/alien/resin/flower_bud/Destroy()
 	QDEL_LIST(vines)
+	QDEL_NULL(countdown)
+	if(spawner) // anti harddel checks
+		spawner.flower_bud = null
+		if(!QDELETED(spawner))
+			qdel(spawner)
+		spawner = null
```

Comment:
Why do we need to clear the flower_bud variable? shouldn't qdeling the spawner do that?